pkg/client/clientset/versioned/clientset.go

```diff
@@ -28,8 +28,6 @@ import (
 type Interface interface {
 	Discovery() discovery.DiscoveryInterface
 	SubmarinerV1() submarinerv1.SubmarinerV1Interface
-	// Deprecated: please explicitly pick a version if possible.
```

Comment:
This shouldn’t be part of this patch.

Reply:
Sorry for the noise, this should not be here.